sqlx-core/src/sqlite/options/mod.rs

```diff
@@ -228,7 +228,9 @@ impl SqliteConnectOptions {
         K: Into<Cow<'static, str>>,
         V: Into<Cow<'static, str>>,
     {
+        self.pragmas.reverse();
         self.pragmas.insert(key.into(), value.into());
+        self.pragmas.reverse();
```

Comment:
@parazyd can you explain why this is necessary? Preferably in a comment in the code here?

Reply:
Added and rebased into one commit.